src/websocket-events/websocket-events.base.ts

```diff
@@ -86,6 +94,9 @@ export abstract class WebSocketEventsBase
     client.on('error', err => {
       this.logger.log(`WebSocket ${client.id}: error: ${err}`);
     });
+    client.on('message', msg => {
+      this.logger.debug(`WS => ${msg}`);
```

Comment:
Do we want to dump all websocket messages?

Reply:
We could switch this to TRACE level. I think that's consistent with other FF microservices